frontend/src/core/comments/components/AddComment.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+
+import * as React from 'react';
+import { Localized } from '@fluent/react';
+
+import './Comment.css';
+
+import { UserAvatar } from 'core/user'
+
+type Props = {|
+    user: string,
+    username: string,
+    imageURL: string,
+|};
+
+
+export default function AddComments(props: Props) {
+    const { user, username, imageURL } = props;
+
+    if (!user) {
+        return null;
+    }
+
+    const submitComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        let comment=document.querySelector('#comment-input').value
+        document.getElementById('comment-input').value = ''
+        console.log(comment);
+    };
+
+    const rejectWithComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        console.log('Reject');
+    };
+
+    const approveWithComment = (event: SyntheticMouseEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        console.log('Approve');
+    };
+
+    return <div className='comment add-comment'>
+        <UserAvatar
+            user={ user }
+            username={ username }
+            title=''
+            imageUrl={ imageURL }
+        />
+        <form className='container'>
+            <div className='content'>
+                <Localized
+                    id='comments-AddComment--input'
+                    attrs={{ placeholder: true }}
+                >
+                    <input
+                        type='text'
+                        id='comment-input'
+                        name='comment'
+                        placeholder={'Write a comment...'}
+                    />
+                </Localized>
+            </div>
+            <div className='options'>
+                <Localized id='comments-AddComment--comment-button'>
+                    <button
+                        className='comment-btn'
+                        onClick={ (e) => submitComment(e) }
+                    >
+                        Comment
+                    </button>
+                </Localized>
+                <Localized id='comments-AddComment--reject-button'>
```

Comment:
I would ignore Reject/Approve & Comment buttons for now, since they'll require quite some effort. We'd need logic to decide when and to whom to show or not show them, and this PR is already getting big.

I'd even be fine with shipping comments without this feature in the first version.